index.html

```diff
@@ -20,21 +20,22 @@
         }
     </style>
 </head>
-<body>
+<body background = yellow>
```

Comment:
This doesn't works because we are using bootstrap, feel free to remove it.
Also it believe it should be `<body bgcolor="yellow">`

Reply:
i did it